### PR TITLE
W2 - anhquach - create recycler view

### PR DESCRIPTION
**Pull request by at-anhquach**
#### Related issues (Optional)
Task 3

#### Description (Required)
- [x] create recycler view

### CODE REVIEW

File: app/src/at-anhquach/java/vn/asiantech/internship/RecyclerViewActivity.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        initData();
+        initAdapter();
+    }
+
+    private void initViews() {
+        mRecyclerViewNewFeed = findViewById(R.id.recyclerViewPerson);
+    }
+
+    private void initData() {
+        mNewFeedList.add(new NewFeed("Hana Hana", "How to complete this issue? How to make it so perfect? How to complete this issue? How to make it so perfect? " +
```
Comment (at-dev-support-01) [inline app/src/at-anhquach/java/vn/asiantech/internship/RecyclerViewActivity.java]: WARNING: [AvoidDuplicateLiterals] The String literal "Hana Hana" appears 7 times in this file; the first occurrence is on line 33
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

File: app/src/at-anhquach/java/vn/asiantech/internship/RecyclerViewActivity.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                "How to complete this issue? How to make it so perfect?", 0));
+        mNewFeedList.add(new NewFeed("Hana Hana", "How to complete this issue?", 0));
```
Comment (at-dev-support-01) [inline app/src/at-anhquach/java/vn/asiantech/internship/RecyclerViewActivity.java]: WARNING: [AvoidDuplicateLiterals] The String literal "How to complete this issue?" appears 6 times in this file; the first occurrence is on line 35
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

**Comment by at-anhquach** [comment]
@at-huongnguyen I completed this issue, please review for me1 Thank you!

File: app/src/at-anhquach/res/layout/item_newfeed.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="10dp"
+            android:ellipsize="end"
+            android:maxLines="2"
+            android:padding="15dp"
+            apps:text='hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh' />
+    </LinearLayout>
+
+</android.support.v7.widget.CardView>
```
Comment (at-huongnguyen) [inline app/src/at-anhquach/res/layout/item_newfeed.xml]: Add line break

**Comment by at-huongnguyen** [comment]
@at-anhquach Please resolve conflicts

**Pull request merged**